### PR TITLE
Add missing dependencies

### PR DESCRIPTION
**Pull request by cortopy**
When running strapi with yarn 2, missing dependencies are reported for this package. This PR adds them

### CODE REVIEW

File: packages/core/admin/package.json [hunk 1]
```diff
@@ -134,6 +136,7 @@
     "webpack-cli": "4.9.1",
     "webpack-dev-server": "4.7.3",
     "webpackbar": "5.0.0-3",
+    "whatwg-fetch": "3.6.2"
```
Comment (soupette) [inline packages/core/admin/package.json:139]: I think this one can be removed as it is a polyfill and browsers support it natively.

Instead can you remove this from the webpack.alias.js file and remove the dependency int @strapi/helper-plugin package?

File: packages/core/admin/package.json [hunk 2]
```diff
@@ -101,6 +101,7 @@
     "mini-css-extract-plugin": "2.4.4",
     "moment": "^2.29.1",
     "node-polyfill-webpack-plugin": "1.1.4",
+    "object-assign": "4.1.1",
```
Comment (soupette) [inline packages/core/admin/package.json:104]: This one is a polyfill as well so we might not need this one. Do you know which lib needs it?
Could you remove it from the webpack.alias.js file?

**Review by soupette — CHANGES_REQUESTED** [review]

File: packages/core/admin/package.json (hunk 2 above)
Comment (cortopy) [reply packages/core/admin/package.json]: I don't now where it comes from. I made the changes although I then bumped into many other errors when building the UI, so it's likely yarn 2 support is not possible for now

File: packages/core/helper-plugin/package.json [hunk 3]
```diff
@@ -58,8 +58,7 @@
     "react-intl": "5.20.2",
     "react-router": "^5.2.0",
     "react-router-dom": "5.2.0",
-    "styled-components": "^5.2.3",
-    "whatwg-fetch": "^3.6.2"
```
Comment (soupette) [inline packages/core/helper-plugin/package.json:62]: This lib is still imported in the helper-plugin/lib/utils/request can you remove the import as well?

**Review by soupette — CHANGES_REQUESTED** [review]

File: packages/core/helper-plugin/package.json (hunk 3 above)
Comment (cortopy) [reply packages/core/helper-plugin/package.json:62]: sorry it took this long. Just done this change

File: packages/core/admin/package.json (hunk 2 above)
Comment (MrChrisRodriguez) [reply packages/core/admin/package.json]: @cortopy we're using Yarn 2 with PnP and Strapi. We do get all of the module warnings, but we're able to build and run successfully. What errors were you running into?

If this PR is dead in the water, my team may take a stab at updating it. Yarn PnP is amazing, and explicitly naming missing dependencies would be great for Strapi

**Comment by derrickmehaffy** [comment]
@cortopy can you resolve the conflicts

**Comment by cortopy** [comment]
> @cortopy can you resolve the conflicts

done

**Comment by gu-stav** [comment]
Hey @cortopy. Apologies for the long delay - we have been working on another feature lately. Based on your PR I did more cleanup of the webpack alias file in https://github.com/strapi/strapi/pull/13606. Furthermore: we have decided to go through all dependency warnings in the strapi packages early next week to fix even more.

I'll therefore close this PR.

Thanks for your work and responsed. 🚀

**Comment by MrChrisRodriguez** [comment]
@gu-stav PR #13513 claims to have reviewed and added missing dependencies on all Strapi packages. You may want to look at that before proceeding with adding the packages manually yourself.

Thanks again @cortopy for helping push Strapi towards solving this problem.

**Pull request closed without merging**